src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -281,8 +299,12 @@ def values(self) -> ArrayLike:
         >>> idx.values
         array([1, 2, 3])
         """
-        # TODO: SNOW-1458117 implement values
-        return self.to_pandas().values
+        return (
+            self._query_compiler.reset_index()
+            .drop(columns=self._query_compiler.columns)
+            .to_numpy()
+            .reshape(-1)
```

Comment:
what is the reshape of -1 doing here? can you add a comment?

Reply:
The reshape simply puts it into a "row vector" like native pandas. It changes shape from (3,) to (3, 1) for example